### PR TITLE
Announce alpakka team

### PR DESCRIPTION
**Pull request by ennru**
Targeted to be published on Wednesday.

### CODE REVIEW

File: blog/_posts/2018-05-02-alpakka-team.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: Lightbend to ramp up Alpakka
+author: Enno Runne
+short: We're happy to announce that Lightbend ramps up Alpakka with its own team.
+category: news
+tags: [alpakka,streams]
+---
+
+Dear hakkers,
+
+One and a half years ago the Akka Team [announced](http://akka.io/blog/2016/08/23/intro-alpakka) a place for the community to collect endpoints for systems integration with Akka Streams: [Alpakka](https://developer.lightbend.com/docs/alpakka/current/).
```
Comment (johanandren) [inline blog/_posts/2018-05-02-alpakka-team.md]: https for all links

File: blog/_posts/2018-05-02-alpakka-team.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+The community showed its strength and we've received a lot of great contributions to this effort to enable enterprise integration in a reactive way.
+
+During this time the Akka Team has enjoyed this great interest to contribute to Alpakka, but did not get to review Pull Requests as often as they would have liked. Core Akka and exploration of useful additions to it has been their main focus.
+
+Therefore, we are very happy to announce that the company behind Akka, [Lightbend Inc.](https://www.lightbend.com/), has set up a separate team to keep after Alpakka and give the community around it the attention it deserves.
```
Comment (ktoso) [inline blog/_posts/2018-05-02-alpakka-team.md]: remove the Inc., makes us sound like some evil corporation  ;-)

File: blog/_posts/2018-05-02-alpakka-team.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+The Alpakka team is now responsible for the [Alpakka repository](https://github.com/akka/alpakka) and the [Akka Streams Kafka repository](https://github.com/akka/reactive-kafka), which is considered part of the Alpakka project.
+
+
+## What does this mean for the community?
+
+The collection of Akka Streams connectors Alpakka is and will continue to be the Open Source project relying on the community.
```
Comment (ktoso) [inline blog/_posts/2018-05-02-alpakka-team.md]: that Alpakka is, will continue to be...

File: blog/_posts/2018-05-02-alpakka-team.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+Feedback on issues and pull requests will be given much faster by the Alpakka team so that the turn-around to implement features and to resolve issues will vastly improve.
+
+
+## What does this mean for Lightbend subscription customers?
+
+Lightbend's intention is to become able to offer support for some parts of Alpakka. That is why the Alpakka team will work to improve the existing connectors and test them more thoroughly.
+Lightbend subscription customers will continue to get support for the Akka Streams Kafka library.
```
Comment (ktoso) [inline blog/_posts/2018-05-02-alpakka-team.md]: I would add a summary like:

```

In other words, by having a full time team responsible for those connectors, you can expect improved responsiveness, quality and commercial support for the Alpakka family of streaming connectors and patterns.
```

File: blog/_posts/2018-05-02-alpakka-team.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+
+We hope you're looking forward to Alpakka becoming an even more lively project, where the community engages to *connect all the things*.
+
+
+Happy hakking!
+
+– The Akka Team and the Alpakka Team
```
Comment (ktoso) [inline blog/_posts/2018-05-02-alpakka-team.md]: nice :)

Comment (ktoso) [reply blog/_posts/2018-05-02-alpakka-team.md]: Maybe `Akka & Alpakka Teams`?

**Pull request merged**